rust/src/dcerpc/log.rs

```diff
@@ -69,11 +69,9 @@ fn log_dcerpc_header_tcp(
         jsb.set_string("response", "UNREPLIED")?;
     }
 
-    if let Some(ref hdr) = state.header {
-        jsb.set_uint("call_id", tx.call_id as u64)?;
-        let vstr = format!("{}.{}", hdr.rpc_vers, hdr.rpc_vers_minor);
-        jsb.set_string("rpc_version", &vstr)?;
-    }
+    jsb.set_uint("call_id", tx.call_id as u64)?;
+    let version = [(tx.version / 10).to_string(), (tx.version % 10).to_string()];
+    jsb.set_string("rpc_version", &version.join("."))?;
```

Comment:
@jasonish is this the rust way to do it ? Is there not a sprintf like ?

Reply:
This is def the python way 😛

Reply:
I believe I could do what was done before here w `fmt`, that'll be the snprintf like way :)